plugins/anchor-links/index.test.js

```diff
@@ -119,43 +207,48 @@ describe('anchor-links', () => {
         ].join('\n')
       )
       expect(headings).toMatchInlineSnapshot(`
-      Array [
-        Object {
-          "aliases": Array [],
-          "level": 1,
-          "permalinkSlug": "hello-world",
-          "slug": "hello-world",
-          "title": "hello world",
-        },
-        Object {
-          "aliases": Array [],
-          "level": 1,
-          "permalinkSlug": "hello-world-1",
-          "slug": "hello-world-1",
-          "title": "hello world",
-        },
-        Object {
-          "aliases": Array [],
-          "level": 1,
-          "permalinkSlug": "foo",
-          "slug": "foo",
-          "title": "foo",
-        },
-        Object {
-          "aliases": Array [],
-          "level": 1,
-          "permalinkSlug": "hello-world-2",
-          "slug": "hello-world-2",
-          "title": "hello world",
-        },
-        Object {
-          "aliases": Array [],
-          "level": 1,
-          "permalinkSlug": "foo-1",
-          "slug": "foo-1",
-          "title": "foo",
-        },
-      ]
+        Array [
```

Comment:
Rest of this is auto-formatting via `prettier`, and addition of `tabbedSectionDepth` to inline snapshots.